<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Main_QMainWindow</class>
	<widget class="QMainWindow" name="Main_QMainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>776</width>
				<height>528</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Home Assessment - Caner Ã–zdemir</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>:/Window/Icon.svg</normaloff>
				:/Window/Icon.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Main_QWidget">
			<property name="font">
				<font />
			</property>
			<layout class="QHBoxLayout" name="horizontalLayout_6">
				<item>
					<widget class="QWidget" name="Manipulation_QWidget" native="true">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Fixed" vsizetype="Preferred">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>110</width>
								<height>0</height>
							</size>
						</property>
						<layout class="QVBoxLayout" name="verticalLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Tool_QWidget" native="true">
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>30</height>
										</size>
									</property>
									<property name="maximumSize">
										<size>
											<width>16777215</width>
											<height>30</height>
										</size>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="PenTool" name="Pen_QPushButton">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="toolTip">
													<string>Pen</string>
												</property>
												<property name="icon">
													<iconset resource="../resource/resource.qrc">
														<normaloff>:/Button/Pen.svg</normaloff>
														:/Button/Pen.svg
													</iconset>
												</property>
												<property name="iconSize">
													<size>
														<width>20</width>
														<height>20</height>
													</size>
												</property>
												<property name="checkable">
													<bool>true</bool>
												</property>
												<property name="checked">
													<bool>true</bool>
												</property>
												<attribute name="buttonGroup">
													<string notr="true">Tool_QButtonGroup</string>
												</attribute>
											</widget>
										</item>
										<item>
											<widget class="QWidget" name="Tool_QWidget_Space" native="true">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
											</widget>
										</item>
										<item>
											<widget class="EraserTool" name="Eraser_QPushButton">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="toolTip">
													<string>Eraser</string>
												</property>
												<property name="icon">
													<iconset resource="../resource/resource.qrc">
														<normaloff>:/Button/Eraser.svg</normaloff>
														:/Button/Eraser.svg
													</iconset>
												</property>
												<property name="iconSize">
													<size>
														<width>20</width>
														<height>20</height>
													</size>
												</property>
												<property name="checkable">
													<bool>true</bool>
												</property>
												<attribute name="buttonGroup">
													<string notr="true">Tool_QButtonGroup</string>
												</attribute>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Settings_QWidget_1" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_2">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="PenSize_QLabel">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>16777215</width>
														<height>30</height>
													</size>
												</property>
												<property name="text">
													<string>Size :</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QWidget" name="Settings_QWidget_1_Space" native="true">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
											</widget>
										</item>
										<item>
											<widget class="PenSize" name="PenSize_QDoubleSpinBox">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>60</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>60</width>
														<height>30</height>
													</size>
												</property>
												<property name="alignment">
													<set>Qt::AlignCenter</set>
												</property>
												<property name="decimals">
													<number>3</number>
												</property>
												<property name="maximum">
													<double>1024.000000000000000</double>
												</property>
												<property name="singleStep">
													<double>0.100000000000000</double>
												</property>
												<property name="value">
													<double>0.000000000000000</double>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Settings_QWidget_2" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_3">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="PenColor_QLabel">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>16777215</width>
														<height>30</height>
													</size>
												</property>
												<property name="text">
													<string>Color :</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QWidget" name="Settings_QWidget_2_Space" native="true">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
											</widget>
										</item>
										<item>
											<widget class="PenColor" name="PenColor_QPushButton">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>30</width>
														<height>30</height>
													</size>
												</property>
												<property name="iconSize">
													<size>
														<width>20</width>
														<height>20</height>
													</size>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Settings_QWidget_3" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_4">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="PenOpacity_QLabel">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>16777215</width>
														<height>30</height>
													</size>
												</property>
												<property name="text">
													<string>Opacity :</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QWidget" name="Settings_QWidget_3_Space" native="true">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
											</widget>
										</item>
										<item>
											<widget class="PenOpacity" name="PenOpacity_QSpinBox">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>55</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>55</width>
														<height>30</height>
													</size>
												</property>
												<property name="alignment">
													<set>Qt::AlignCenter</set>
												</property>
												<property name="maximum">
													<number>255</number>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Manipulation_QWidget_Space" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Settings_QWidget_4" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_5">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="Zoom_QLabel">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>16777215</width>
														<height>30</height>
													</size>
												</property>
												<property name="text">
													<string>Zoom :</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QWidget" name="Settings_QWidget_4_Space" native="true">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Expanding" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
											</widget>
										</item>
										<item>
											<widget class="Zoom" name="Zoom_QDoubleSpinBox">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>60</width>
														<height>30</height>
													</size>
												</property>
												<property name="maximumSize">
													<size>
														<width>60</width>
														<height>30</height>
													</size>
												</property>
												<property name="alignment">
													<set>Qt::AlignCenter</set>
												</property>
												<property name="decimals">
													<number>3</number>
												</property>
												<property name="maximum">
													<double>8.000000000000000</double>
												</property>
												<property name="singleStep">
													<double>0.050000000000000</double>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QScrollArea" name="Image_QScrollArea">
						<property name="cursor" stdset="0">
							<cursorShape>CrossCursor</cursorShape>
						</property>
						<widget class="PaintArea" name="Image_QWidget">
							<property name="geometry">
								<rect>
									<x>0</x>
									<y>0</y>
									<width>640</width>
									<height>480</height>
								</rect>
							</property>
							<property name="sizePolicy">
								<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
									<horstretch>0</horstretch>
									<verstretch>0</verstretch>
								</sizepolicy>
							</property>
							<property name="minimumSize">
								<size>
									<width>30</width>
									<height>30</height>
								</size>
							</property>
						</widget>
					</widget>
				</item>
			</layout>
		</widget>
		<widget class="MenuBar" name="Main_QMenuBar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>776</width>
					<height>21</height>
				</rect>
			</property>
			<property name="font">
				<font>
					<pointsize>9</pointsize>
				</font>
			</property>
			<property name="nativeMenuBar">
				<bool>false</bool>
			</property>
		</widget>
		<action name="Open_QAction">
			<property name="text">
				<string>Open</string>
			</property>
		</action>
	</widget>
	<customwidgets>
		<customwidget>
			<class>PenColor</class>
			<extends>QPushButton</extends>
			<header>component/pencolor.h</header>
		</customwidget>
		<customwidget>
			<class>PenSize</class>
			<extends>QDoubleSpinBox</extends>
			<header>component/pensize.h</header>
		</customwidget>
		<customwidget>
			<class>PaintArea</class>
			<extends>QWidget</extends>
			<header>component/paintarea.h</header>
		</customwidget>
		<customwidget>
			<class>PenOpacity</class>
			<extends>QSpinBox</extends>
			<header>component/penopacity.h</header>
		</customwidget>
		<customwidget>
			<class>PenTool</class>
			<extends>QPushButton</extends>
			<header>component/pentool.h</header>
		</customwidget>
		<customwidget>
			<class>EraserTool</class>
			<extends>QPushButton</extends>
			<header>component/erasertool.h</header>
		</customwidget>
		<customwidget>
			<class>Zoom</class>
			<extends>QDoubleSpinBox</extends>
			<header>component/zoom.h</header>
		</customwidget>
		<customwidget>
			<class>MenuBar</class>
			<extends>QMenuBar</extends>
			<header>component/menubar.h</header>
		</customwidget>
	</customwidgets>
	<resources>
		<include location="../resource/resource.qrc" />
	</resources>
	<connections>
		<connection>
			<sender>Pen_QPushButton</sender>
			<signal>toggled(bool)</signal>
			<receiver>Settings_QWidget_2</receiver>
			<slot>setVisible(bool)</slot>
			<hints>
				<hint type="sourcelabel">
					<x>23</x>
					<y>53</y>
				</hint>
				<hint type="destinationlabel">
					<x>64</x>
					<y>133</y>
				</hint>
			</hints>
		</connection>
	</connections>
	<buttongroups>
		<buttongroup name="Tool_QButtonGroup" />
	</buttongroups>
</ui>